panel/models/chatarea_input.ts

```diff
@@ -31,9 +32,15 @@ export class ChatAreaInputView extends PnTextAreaInputView {
     super.render()
 
     this.el.addEventListener("keydown", (event) => {
-      if (event.key === "Enter" && !event.shiftKey) {
+      console.log( "DBG: keydown event", event.key )
+      if (event.key === "Enter" && (  (!event.shiftKey &&  this.model.enter_sends)
+                                   || ( event.shiftKey && !this.model.enter_sends))
```

Comment:
As a suggestion, this xor can be written more concisely as:
```suggestion
      if (event.key === "Enter" && event.shiftKey != this.model.enter_sends)
```

Reply:
Hmm, nice! It will need a comment though: I had to really stare at it
to figure out what it does  :-)